jd4/spj.py

```diff
@@ -0,0 +1,165 @@
+import pyximport; pyximport.install()
```

Comment:
please don't duplicate code. prefer to add these into case.py . also could you create a "modern case" (might need to think about the naming), correspond to the "legacy case", and make spj optional (to support configuration)?